transformer_engine/pytorch/csrc/comm_gemm_overlap.h

```diff
@@ -192,12 +187,17 @@ struct UbufCommOverlap : torch::CustomClassHolder, UbufBase {
       printf("!!! [UB] Register UBuf %d\n", _ub_reg);
     }
 
+    // if priorities are zero, use max priority for comm and min priority for GEMM
+    if (gemm_priority == 0 && comm_priority == 0)
+      std::tie(gemm_priority, comm_priority) = transformer_engine::cuda::stream_priority_range();
```

Comment:
Huh? How can we use those functions in the framework plugin? We should not be exposing C++ symbols from transformer_engine.so. Most probably (for now, until the refactor) those functions should be part of this extension.

Reply:
Thanks for the heads up! I'm replacing this with a pure C signature.